src/histogram.js

```diff
@@ -196,16 +196,16 @@ angular.module('grafana.directives').directive('grafanaHistogram', function($roo
           if(filterMin && currentValue < minValue) continue;
           if(filterMax && currentValue > maxValue) continue;
 
-          var bucket = (Math.floor(currentValue / bucketSize)*bucketSize).toFixed(3);
+          var bucket = Math.floor(currentValue / bucketSize) * bucketSize;
           if (bucket in values) {
             values[bucket]++;
           } else {
             values[bucket] = 1;
           }
         }
-        _.forEach(Object.keys(values).sort(), function(key) {
+        for(var key in values) {
```

Comment:
> As I understand dictionary in JS should be sorted by design (at least this is true for Chrome)

I check the behavior, and found that it's not true inf Firefox.
https://jsfiddle.net/fy5tv8ys/2/